tests/edamam/test_edamam_pipeline.py

```diff
@@ -0,0 +1,39 @@
+from datetime import datetime
+import pytest
+
+import dlt
+
+from pipelines.edamam import edamam_source, nutrition_data
+
+from tests.utils import ALL_DESTINATIONS, assert_load_info
+
+@pytest.mark.parametrize('destination_name', ALL_DESTINATIONS)
+def test_load_nutrition_data(destination_name: str) -> None:
+    # create the pipeline
+    pipeline = dlt.pipeline(
+        pipeline_name="edamam",
+        destination=destination_name,
+        dataset_name="edamam_data",
+        full_refresh=True
+    )
+    ingredients = ['100 gr chicken']
+    info = pipeline.run(nutrition_data(ingredients))
+    # test if jobs is loaded
+    assert_load_info(info)
+    # assert the schema
+    table = pipeline.default_schema.data_tables()
+    assert len(table) == 1
+    # assert that table is a dict
+    assert isinstance(table[0], dict)
```

Comment:
the test is good. but you can reuse some building blocks. see tests/utils.py. there's a method that will return you row counts for tables. you also do not need to check the datatypes in the destination. do it by inspecting dlt schema, you already get table here. just check the columns.